engine/table/src/main/java/io/deephaven/engine/table/impl/updateby/rollingformulamulticolumn/windowconsumer/ObjectRingBufferWindowConsumer.java

```diff
@@ -0,0 +1,43 @@
+//
+// Copyright (c) 2016-2024 Deephaven Data Labs and Patent Pending
+//
+package io.deephaven.engine.table.impl.updateby.rollingformulamulticolumn.windowconsumer;
+
+import io.deephaven.base.ringbuffer.ObjectRingBuffer;
+import io.deephaven.chunk.ObjectChunk;
+import io.deephaven.chunk.Chunk;
+
+public class ObjectRingBufferWindowConsumer<T> extends RingBufferWindowConsumer {
+    private final ObjectRingBuffer<T> objectRingBuffer;
+
+    private ObjectChunk<T, ?> influencerValuesChunk;
+
+    public ObjectRingBufferWindowConsumer(ObjectRingBuffer<T> objectRingBuffer) {
+        this.objectRingBuffer = objectRingBuffer;
```

Comment:
Same comments as CharRingBuffer, mentioning here since this class is not replicated.